FluentUI.Drawer/src/main/java/com/microsoft/fluentui/persistentbottomsheet/sheetItem/VerticalViewProvider.kt

```diff
@@ -36,7 +36,7 @@ internal class VerticalViewProvider(val context: Context) : IViewProvider {
             val listener = this
             verticalItemAdapter.onBottomSheetItemClickListener = object : OnClickListener {
                 override fun onBottomSheetItemClick(item: BottomSheetItem) {
-                    listener.onSheetItemClick(SheetItem(item.id, item.imageId, item.title, item.customBitmap))
+                    listener.onSheetItemClick(SheetItem(item.id, item.title, item.imageId, item.imageTint, item.customBitmap))
```

Comment:
Is the tint always here?  In getVerticalItemlist, we have checked for null in case of tint.
Please verify the same

Reply:
if it is null, same will be returned to caller, So here it would be same sheet item passed to consumer app.